src/index.ts

```diff
@@ -64,21 +65,8 @@ module.exports = class ServerlessPlugin {
     this.serverless.cli.log("Auto instrumenting functions with Datadog");
     const config = getConfig(this.serverless.service);
     setEnvConfiguration(config, this.serverless.service);
-    const defaultRuntime = this.serverless.service.provider.runtime;
-    let defaultNodeRuntime: RuntimeType.NODE | RuntimeType.NODE_ES6 | RuntimeType.NODE_TS | undefined;
-    switch (config.nodeModuleType) {
-      case "es6":
-        defaultNodeRuntime = RuntimeType.NODE_ES6;
-        break;
-      case "typescript":
-        defaultNodeRuntime = RuntimeType.NODE_TS;
-        break;
-      case "node":
-        defaultNodeRuntime = RuntimeType.NODE;
-        break;
-    }
 
-    const handlers = findHandlers(this.serverless.service, defaultRuntime, defaultNodeRuntime);
+    const handlers = this.getHandlers(config.nodeModuleType);
     if (config.addLayers) {
       this.serverless.cli.log("Adding Lambda Layers to functions");
       this.debugLogHandlers(handlers);
```

Comment:
I left this unchanged as of right now, but two lines below this one here has:
```
if (hasWebpackPlugin(this.serverless.service)) {
        forceExcludeDepsFromWebpack(this.serverless.service);
}
```
I think we may have originally talked about removing this logic, but wanted to double check.